FirebaseAuth/Tests/Unit/FIRAuthTests.m

```diff
@@ -1513,6 +1521,39 @@ - (void)testSignInWithCustomTokenSuccess {
   OCMVerifyAll(_mockBackend);
 }
 
+/** @fn testSignInWithCustomTokenWithoutRefreshTokenSuccess
+    @brief Tests the flow of a successful @c signInWithCustomToken:completion: call without a
+   refresh token.
+ */
+- (void)testSignInWithCustomTokenWithoutRefreshTokenSuccess {
```

Comment:
Could you add a test to assert the intended error occurs when it is _not_ successful? I believe it should be the error with the `kRefreshTokenError` message.

Reply:
Hi Nick,
Thank you for suggesting to add more tests. Added `testGetIDTokenResultForcingRefreshFailureNoRefreshToken` in `FIRUsersTests.m` to test the scenario when a user tries to refresh without a refresh token. However I don't know the syntax to verify NSError with message `kRefreshTokenError` yet: I am still pretty new to iOS development. Will keep digging. If you know an existing example, please do point me to it.

Reply:
Added syntax to verify the correct NSError is returned from SecureTokenService. Mark this as resolved.